config/path

```diff
@@ -200,8 +200,12 @@ if [ -z "$PATH" -o "$PATH" = "${PATH#$ROOT/$TOOLCHAIN/bin:}" ]; then
   export PATH="$ROOT/$TOOLCHAIN/bin:$ROOT/$TOOLCHAIN/sbin:$PATH"
 fi
 
-if [ -z "$LD_LIBRARY_PATH" -o "$LD_LIBRARY_PATH" = "${LD_LIBRARY_PATH#$ROOT/$TOOLCHAIN/lib:}" ]; then
-  export LD_LIBRARY_PATH="$ROOT/$TOOLCHAIN/lib:$LD_LIBRARY_PATH"
+if [ -z "$LD_LIBRARY_PATH" ]; then
+  export LD_LIBRARY_PATH="$ROOT/$TOOLCHAIN/lib"
+else
+  if [ "$LD_LIBRARY_PATH" = "${LD_LIBRARY_PATH#$ROOT/$TOOLCHAIN/lib:}" ]; then
```

Comment:
> the tailing : (separator) is needed in comparison.

Right, but when `$LD_LIBRARY_PATH == $ROOT/$TOOLCHAIN/lib` (ie. without the trailing colon) won't you just end up setting `LD_LIBRARY_PATH=$ROOT/$TOOLCHAIN/lib:$LD_LIBRARY_PATH` in line 207, because there is no trailing : and the comparison is true (because the substitution doesn't remove anything, as there is no colon) which then results in `LD_LIBRARY_PATH=$ROOT/$TOOLCHAIN/lib:$ROOT/$TOOLCHAIN/lib`?

Removing the : makes the comparison less safe, as you can't tell the difference between the lib directory and a subdirectory of lib...

Is the presence of the trailing colon actually causing a problem?